net-print/cnijfilter-drivers/cnijfilter-drivers-3.80.ebuild

```diff
@@ -0,0 +1,167 @@
+# Copyright 1999-2012 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+
+MY_PN="${PN/-drivers/}"
+
+inherit eutils autotools flag-o-matic multilib
+
+DESCRIPTION="Canon InkJet Printer Driver for Linux (Pixus/Pixma-Series)."
+HOMEPAGE="http://support-sg.canon-asia.com/contents/SG/EN/0100469302.html"
+SRC_URI="http://gdlp01.c-wss.com/gds/3/0100004693/01/${MY_PN}-source-${PV}-1.tar.gz"
```

Comment:
You're using the filename twice (here and in S), so put it into `MY_P` variable instead. You can also kill `MY_PN` since it is of no value as separate variable.

Reply:
Fixed